autopilot/contractor.go

```diff
@@ -1612,16 +1614,16 @@ func endHeight(cfg api.AutopilotConfig, currentPeriod uint64) uint64 {
 
 // renterFundsToExpectedStorage returns how much storage a renter is expected to
 // be able to afford given the provided 'renterFunds'.
-func renterFundsToExpectedStorage(renterFunds types.Currency, duration uint64, host rhpv2.HostSettings) uint64 {
-	costPerByte := host.UploadBandwidthPrice.Add(host.StoragePrice.Mul64(duration)).Add(host.DownloadBandwidthPrice)
-	// If storage is free, we can afford 'unlimited' data.
-	if costPerByte.IsZero() {
-		return math.MaxUint64
+func renterFundsToExpectedStorage(renterFunds types.Currency, duration uint64, settings rhpv2.HostSettings, pt rhpv3.HostPriceTable) uint64 {
```

Comment:
should we use the pricetable for our cost calculations in `renewFundingEstimate` as well?